### ISSUE TITLE
next export interferes with next start

### ISSUE THREAD

**Issue by kanishka-work** [vercel/next.js#26027]
### What version of Next.js are you using?

10

### What version of Node.js are you using?

14

### What browser are you using?

firefox

### What operating system are you using?

ubuntu

### How are you deploying your application?

vercel

### Describe the Bug

If you create a next site, add a single file "test.html" in the public directory, run next build, next export, and then observe the deployed site, the file is accessible at the route "test". If you don't run next export, then the file is accessible at the route "test.html"

### Expected Behavior

I don't expect next export to have any impact on next start and/or interfere with the results of next build.

### To Reproduce

See description

**Comment by timneutkens** [comment]
Can you create a reproduction? It's hard to follow what exactly you're running when deploying.

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
